src/components/StationsList.jsx

```diff
@@ -0,0 +1,27 @@
+import { array } from 'prop-types';
+import React from 'react';
+import Button from './Button';
+
+export const StationsList = ({ stations, handleSelect, ...props }) => {
+  if (!stations || stations.length === 0) {
+    const textError =
+      'Your stations array is empty! You need to have at least one station to create the StationsList component.';
+    throw new Error(textError);
+  }
```

Comment:
Don't throw if stations list is empty, instead return an empty `<ul>` - which will be done automatically as `stations.map` will render nothing